Makefile

```diff
@@ -33,7 +33,9 @@ minicriu.o : CFLAGS += -fPIE
 
 minicriu-client.o : CFLAGS += -fPIC
 
-libminicriu-client.a : minicriu-client.o
+cleanup.o : CFLAGS += -fPIC
```

Comment:
Should it be a separate module? It has to export function names -- something that I would like to avoid. Or we may need to introduce additional partial linking -- would be unnecessary if there will be no new files.